crates/sc-subspace-block-relay/src/consensus.rs

```diff
@@ -241,9 +241,9 @@ where
                 Ok(Ok(downloaded))
             }
             Err(error) => {
-                trace!(
+                info!(
```

Comment:
Please don't make this info, I do not think it is appropriate level for these messages. Is it a significant error that is within user's control? If yes this should be `warn` or `error`, if not then at most `debug`.

Reply:
sg, changed to debug